CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,52 @@
+# Contributor Code of Conduct
```

Comment:
@MikeMcC399 Nice! For maintainability, it would be better to update this to follow the approach we have taken with our cypress-documentation repository that points people back to the cypress repo's documentation. For ref: https://github.com/cypress-io/cypress-documentation/blob/main/CODE_OF_CONDUCT.md

This ensures there is one source of truth and that we only need to make the update in one place!

Reply:
@emilyrohrbough 
Good suggestion! I have replaced the text with a link to the main CODE_OF_CONDUCT.md